### DISCUSSION TITLE
does PyG contain GCN implementation on Zachary Karate Dataset?

### DISCUSSION THREAD

**Discussion by utkarsh0902311047** [pyg-team/pytorch_geometric#7983] [Q&A]
I tried to write GCN implementation without using any libraries for Karate Dataset and wanted to match the gradients and final results.
So is there any PyG implementation of the Zachary Karate dataset?

**Comment by rusty1s** [comment]
Yes, see `datasets.KarateClub`. Our tutorial goes over its usage as well, see https://colab.research.google.com/drive/1h3-vJGRVloF5zStxL5I0rSy4ZUPNsjy8?usp=sharing
